source/guides/maintenance/fix-broken-install.rst

```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
 
 #. Ensure the system is connected to the network.
 
-#. Mount the system’s root partition.
+#. Mount the system’s root and boot partitions.
 
-   #. To find the root partition, run:
+   #. To find the root and boot partitions, run:
 
       .. code-block:: bash
 
-         lsblk
+         lsblk -o NAME,LABEL,PARTTYPE,PARTLABEL
 
-      We'll use :file:`/dev/sda3/` as the root partition example.
+      We'll use :file:`/dev/sda3/` as the root partition example and :file:`/dev/sda1/` as the boot partition example.
 
-   #. Next, mount the partition to the :file:`/mnt` folder.
+   #. Next, mount the root partition to the :file:`/mnt` folder and the boot partition to the :file:`/mnt/boot` folder.
```

Comment:
swupd does not manage the EFI partition. So we it's not necessary to mount the boot partition.